internal/inventory/inventory.go

```diff
@@ -29,13 +29,17 @@ import (
 	"github.com/samber/lo"
 )
 
-const indexTemplate = "logs-cloud_asset_inventory.asset_inventory-%s_%s_%s_%s-default"
+const (
+	indexTemplate = "logs-cloud_asset_inventory.asset_inventory-%s_%s_%s_%s-default"
+	minimalPeriod = 30 * time.Second
```

Comment:
maybe can omit this const? 

locally we can tweak 
https://github.com/elastic/cloudbeat/blob/56579fcae8736f2fb6775ff11aa8b2702a9a8e3e/deploy/asset-inventory/cloudbeat-gcp-asset-inventory.yml#L14

and users have a hard-coded value, for example - https://github.com/elastic/integrations/blob/56d0473a2ed4900b7b24b6a254795b9bc82cb713/packages/cloud_asset_inventory/data_stream/asset_inventory/agent/stream/gcp.yml.hbs#L1

Reply:
I would rather keep it. It's not a default value, but a guardian value. If the customer sets value too low, this will start to spawn many goroutines that hit API. I can lower it if you think it makes sense, but we **definitely** need to make sure there is a reasonable minimum threshold. Obviously the worst-case scenario is `period: 0s`, where we get an infinite loop of new goroutines, but even `5s` or `10s` is too small IMHO.

Reply:
PTAL: https://github.com/elastic/cloudbeat/pull/2902/files#diff-151451acd3d777673443e215b546390195bf895e79500c18a34bdf0029222fbfR57-R59

Reply:
np with keeping it imo. but this value isn't configurable by the users, it's hardcoded in the integration package.